Modules/Office365DSC/DSCResources/MSFT_O365AdminAuditLogConfig/MSFT_O365AdminAuditLogConfig.psm1

```diff
@@ -10,8 +10,9 @@ function Get-TargetResource
         $IsSingleInstance,
 
         [Parameter()]
-        [ValidateSet('Present', 'Absent')]
-        [string]$Ensure = 'Present',
+        [ValidateSet('Present')]
+        [string]
```

Comment:
In the Set method you updated the type to System.String. Lets do it here and in the Test method as well to keep it consistent.